### PR TITLE
Fix/1047

### PR DESCRIPTION
**Pull request by Jehoel**
Adds mitigations for issue #1047

* Suppresses XML and `async`/`await` compiler warnings in test projects due to the (literally) thousands of spurious warnings.

* Fixes build errors (not warnings) that are raised when I run the command-line `build\01_Build.bat` on my computer.

I'm not able to fix the errors from `02_CreatePackages.bat`, however.

### CODE REVIEW

**Comment by RicoSuter** [comment]
These scripts are not needed anymore, we just use “dotnet build”. See azure-piplines in the repo root. Is this pr needed then?

File: src/NJsonSchema.CodeGeneration.TypeScript/NJsonSchema.CodeGeneration.TypeScript.csproj [hunk 1]
```diff
@@ -5,7 +5,9 @@
     <Version>10.0.22</Version>
     <PackageTags>json schema validation generator .net</PackageTags>
     <Copyright>Copyright © Rico Suter, 2018</Copyright>
+    <!--
```
Comment (RicoSuter) [inline src/NJsonSchema.CodeGeneration.TypeScript/NJsonSchema.CodeGeneration.TypeScript.csproj]: Also use license type?

Comment (Jehoel) [reply src/NJsonSchema.CodeGeneration.TypeScript/NJsonSchema.CodeGeneration.TypeScript.csproj]: I don't know what the tag is inside `csproj` files, only `nuspec` files.

**Comment by Jehoel** [comment]
> Is this pr needed then?

As it makes the Error List window in VS more readable, and fixes that compile error `src/NJsonSchema.Tests/Generation/InheritanceTests.cs` I like to think so.

**Comment by RicoSuter** [comment]
v10.0.23

**Pull request merged**